phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexToolIT.java

```diff
@@ -237,6 +237,80 @@ public void testWithSetNull() throws Exception {
         }
     }
 
+    @Test
+    public void testIncorrectRebuild() throws Exception {
+        // This test is for building non-transactional mutable global indexes with direct api
+        if (localIndex || transactional || !mutable || !directApi || useSnapshot) {
```

Comment:
Why not immutable?

Reply:
value for included column is changing here.

Reply:
Oh, yes you are right.